solver/src/solver/http_solver/settlement.rs

```diff
@@ -576,8 +518,8 @@ mod tests {
                         {
                             "sell_token": "0xc778417e063141139fce010982780140aa0cd5ab",
                             "buy_token": "0xe4b9895e638f54c3bee2a3a78d6a297cc03e0353",
-                            "exec_sell_amount": "1",
-                            "exec_buy_amount": "2",
+                            "exec_sell_amount": "3",
+                            "exec_buy_amount": "4",
```

Comment:
Any particular reason for this? Or just for fun.

Reply:
So that all AMMs have different input/output values.